index.bs

```diff
@@ -1991,13 +2002,20 @@ Given an [=attribution rate-limit record=] |newRecord|:
      * |record|'s [=attribution rate-limit record/source site=] and |newRecord|'s [=attribution rate-limit record/source site=] are equal
      * |record|'s [=attribution rate-limit record/attribution destination=] and |newRecord|'s [=attribution rate-limit record/attribution destination=] are equal
      * |record|'s [=attribution rate-limit record/time=] is at least [=attribution rate-limit window=] before |newRecord|'s [=attribution rate-limit record/time=]
-1. Let |distinctReportingOrigins| be a new empty [=ordered set=].
-1. [=set/iterate|For each=] |record| of |matchingRateLimitRecords|, [=set/append=] |record|'s
-     [=attribution rate-limit record/reporting origin=] to |distinctReportingOrigins|.
-1. If |distinctReportingOrigins| [=list/contains=] |newRecord|'s
-    [=attribution rate-limit record/reporting origin=], return <strong>allowed</strong>.
-1. If |distinctReportingOrigins|'s [=list/size=] is greater than or equal to |max|, return
-    <strong>blocked</strong>.
+1. Let |distinctReportingOrigins| be the [=set=] of all [=attribution rate-limit record/reporting origin=] in |matchingRateLimitRecords|,
+    [=set/union|unioned=] with «|newRecord|'s [=attribution rate-limit record/reporting origin=]»
+1. If |distinctReportingOrigins|'s [=list/size=] is greater than |max|, return <strong>blocked</strong>.
+
+NOTE: [=rate-limit scope/source=] scopes have an auxiliary [=max source reporting origins per source reporting site=] rate limit that also must be enforced.
+
+1. If |newRecord|'s [=attribution rate-limit record/scope=] is "<code>[=rate-limit scope/attribution=]</code>", return <strong>allowed</strong>
+1. Let |matchingRateLimitRecords| be all [=attribution rate-limit records=] |record| in the [=attribution rate-limit cache=] where all of the following are true:
+     * |record|'s [=attribution rate-limit record/scope=] is "<code>[=rate-limit scope/source=]</code>"
+     * |record|'s [=attribution rate-limit record/source site=] and |newRecord|'s [=attribution rate-limit record/source site=] are equal
+     * |record|'s [=attribution rate-limit record/time=] is at least [=origin rate-limit window=] before |newRecord|'s [=attribution rate-limit record/time=]
```

Comment:
IIUC this is setting matchingRateLimitRecords to be all records before "now - window". 

Don't we instead want this to be all source records within the current window instead (i.e. after "now - window")?

Reply:
Done